group-coordinator/src/main/resources/common/message/ConsumerGroupRegexValue.json

```diff
@@ -0,0 +1,29 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "type": "data",
+  "name": "ConsumerGroupRegexValue",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  "fields": [
+    { "name": "MatchingTopicsNames", "type": "[]string", "versions": "0+",
+      "about": "The list of topic names that match the regular expression." },
+    { "name": "MetadataVersion", "type": "int32", "versions": "0+",
+      "about": "The version of the metadata used to evaluate the regular expression." },
+    { "name": "MemberCount", "type": "int32", "versions": "0+",
+      "about": "The number of members subscribed to the regular expression." }
```

Comment:
We probably don't need it here because we could compute it based on the `ConsumerGroupMemberMetadataValue` record -- the one which contains the subscribed regex.